src/dapps/ens-manager-dapp/handlers/handlerPermanentName.js

```diff
@@ -10,20 +10,22 @@ import { mapGetters, mapState } from 'vuex';
 import { toBN, toHex, fromWei, sha3 } from 'web3-utils';
 import { estimateGasList } from '@/core/helpers/gasPriceHelper.js';
 import { Toast, ERROR } from '@/modules/toast/handler/handlerToast';
-
+import ReverseRegistrar from '@ensdomains/ens-contracts/deployments/mainnet/ReverseRegistrar.json';
 const bip39 = require('bip39');
 
 export default class PermanentNameModule extends ENSManagerInterface {
   constructor(name, address, network, web3, ens, expiry) {
```

Comment:
Make sure this class is getting recreated wherever it's used when web3 is updated as well if you're not using the state reference directly.